app/connectors/EmailConnector.scala

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2025 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package connectors
+
+import config.Service
+import connectors.EmailHttpParser.*
+import logging.Logging
+import models.emails.EmailSendingResult.EMAIL_NOT_SENT
+import models.emails.{EmailSendingResult, EmailToSendRequest}
+import play.api.Configuration
+import play.api.libs.json.Json
+import play.api.libs.ws.writeableOf_JsValue
+import uk.gov.hmrc.http.client.HttpClientV2
+import uk.gov.hmrc.http.{BadGatewayException, GatewayTimeoutException, HeaderCarrier, StringContextOps}
+
+import javax.inject.{Inject, Singleton}
+import scala.concurrent.{ExecutionContext, Future}
+
+
+@Singleton
+class EmailConnector @Inject()(config: Configuration, httpClientV2: HttpClientV2) extends Logging {
+
+  private val baseUrl = config.get[Service]("microservice.services.email")
+
+  def send(email: EmailToSendRequest)
+          (implicit ec: ExecutionContext, hc: HeaderCarrier): Future[EmailSendingResult] = {
+    httpClientV2.post(url"${baseUrl}hmrc/email").withBody(Json.toJson(email)).execute[EmailSendingResult].recover {
```

Comment:
I think due to this being fire and forget, we want to log all non successful error responses as errors, both in the recovers as well in the HttpParser

Reply:
Could you make these `error` level logs?

And I think you need to add logs here as well https://github.com/hmrc/ioss-netp-registration-frontend/pull/22/files#diff-50e86bff226d15637b82ff1c9832b417b5e4159e48d577938df0b62592e21435R32-R34